### DISCUSSION TITLE
Local emissions intensity for Quebec, Canada (QC)

### DISCUSSION THREAD

**Discussion by christopheradlam** [electricitymaps/electricitymaps-contrib#4863] [General]
@pierresegonne I've got another Canada-specific update. I recently had an extensive conversation with Hydro-Quebec on twitter about the two different lifecycle emissions factors they've had published, as they had a report from 2014 that didn't match the current figures on their website. 

They got back to me today and indicated that they had an extensive updated lifecycle emissions analysis performed in 2021, which includes methane release from the reservoirs, which is why the numbers changed. That updated report is this one:
https://www.sciencedirect.com/science/article/pii/S1364032120307206#tbl5

Which resulted in the values you see on this page (maintained by Hydro-Quebec):
https://www.hydroquebec.com/about/our-energy.html
Québec Hydropower: 28
Wind: 14
Nuclear: 8
Photovoltaic Solar: 64
Thermal Natural Gas: 608
Thermal Oil: 880
Thermal Coal: 880

**Comment by pierresegonne** [comment]
These results look pretty solid. 

Not sure though where 28g comes from, it seems that the paper is hinting more at 29.9 or something like that?

Do you have other sources than hydro quebec itself for factors that are not hydro? It's a bit weird that they use the same factor for oil and coal for example?

**Reply by VIKTORVAV99** [reply]
Photovoltaic also appears high?
I don't think it's necessarily wrong but we might want to go with INCER ACV instead?

**Reply by christopheradlam** [reply]
This is the abbreviated version of the previous report:
https://www.hydroquebec.com/data/developpement-durable/pdf/15094A.pdf

This is the whole (previous) report:
https://www.hydroquebec.com/data/developpement-durable/pdf/comparing-power-generation-options-and-electricity-mixes.pdf

A lot is the same between these and the updated one but some is different. Gas is higher, coal and oil are 1 off, but PV, nuclear and wind are all the same. The big change was the increase in HQ's hydro numbers. 

PV CF in Canada is pretty low, so I'm likely not as surprised by that number as you guys are. That said, I will look at the Ontario numbers and see how far off they are.
